### ISSUE TITLE
Combine seam sphere with other seam settings.

### ISSUE THREAD

**Issue by cfytrok** [supermerill/SuperSlicer#3183]
Could you make a detailed description of how the seam sphere works? Is it possible to combine seam sphere with other seam position settings? With seam painting? With Height range modifier?

**Comment by jhlchu** [comment]
> Is it possible to combine seam sphere with other seam position settings?

No. Seam sphere seams seem to override seam Modifiers and "Print Settings" seams. Image below I set seam to corner in "Print Settings". The sphere is seam sphere, white dots are seams. Seam sphere seam locations appear to be a projection from the origin of the sphere onto the closest surface on the object, usually one per layer.
<img src="https://user-images.githubusercontent.com/5092505/188799707-fc58ee5e-b55d-41fe-8ddd-b8db4fce04bc.jpg" width=500 />
<img src="https://user-images.githubusercontent.com/5092505/188799709-2e85089d-ddf0-4fed-8cf7-7f5dac502086.jpg" width=500 />

> With seam painting?

Seam painting can override seam sphere. Order of precedence is as such: Seam painting > Seam sphere > Seam Modifier & Settings
<img src="https://user-images.githubusercontent.com/5092505/188799706-b4049165-f742-421b-bdbb-c4efaf4c7fe0.jpg" width=500 />
<img src="https://user-images.githubusercontent.com/5092505/188799702-3aee86c9-af27-4080-83e6-3f2b4907a530.jpg" width=500/>


> With Height range modifier?

Is there a seam modifier for height range?

### This doesn't seem to be an "Issue". "[Discussions](https://github.com/supermerill/SuperSlicer/discussions)" is probably the appropriate place for this topic.

**Comment by supermerill** [comment]
The center of the sphere attract the seam.
If you put more than one, the one used is the one that is the nearest in the z direction.
